homeassistant/components/mqtt/mixins.py

```diff
@@ -571,6 +584,8 @@ async def async_will_remove_from_hass(self):
     @property
     def available(self) -> bool:
         """Return if the device is available."""
+        if DATA_MQTT not in self.hass.data:
```

Comment:
Not sure this is needed though. But may be some test can probe this is can avoid unneeded errors in the log when the entry is disabled.